qualtran/bloqs/gf_arithmetic/gf2_inverse_test.py

```diff
@@ -53,3 +57,9 @@ def test_gf2_inverse_classical_sim(m):
     bloq = GF2Inverse(m)
     GFM = GF(2**m)
     assert_consistent_classical_action(bloq, x=GFM.elements[1:])
+
```

Comment:
can you add a test for symbolic cost?

Reply:
The test for symbolic cost exists above in `test_gf2_inverse_symbolic_toffoli_complexity`